### PR TITLE
Updated minion and master config files to 2016.11

### PR DESCRIPTION
**Pull request by javierbertoli**
* Added some missing parameters in both `{master,minion}/f_defaults.conf` files
* Kept some backward compatible values (like `hash_type: md5` instead of `sha256`) as changing them to new defaults is too disruptive (require cleaning caches, etc.)

### CODE REVIEW

**Comment by aboe76** [comment]
@nmadhok would you please take a look at see if you are also ok with this then we can merge it...

**Comment by nmadhok** [comment]
@aboe76 Looks good to me! I'll go ahead and merge. Thanks!

File: salt/files/master.d/f_defaults.conf [hunk 1]
```diff
@@ -62,9 +62,23 @@
...
+
 # Directory used to store public key data:
 {{ get_config('pki_dir', '/etc/salt/pki/master') }}
 
+# Key cache. Increases master speed for large numbers of accepted
+# keys. Available options: 'sched'. (Updates on a fixed schedule.)
+# Note that enabling this feature means that minions will not be
+# available to target for up to the length of the maintanence loop
+# which by default is 60s.
+{%- if cfg_minion['key_cache'] in cfg_master %}
```
Comment (aboe76) [inline salt/files/master.d/f_defaults.conf]: I found a small oops...

Comment (aboe76) [inline salt/files/master.d/f_defaults.conf]: should be cfg_master...

Comment (aboe76) [inline salt/files/master.d/f_defaults.conf]: I think it should only be:
```
{{ get_config('key_cache', '') }}
```

**Pull request merged**